main.tex

```diff
@@ -84,42 +84,39 @@ \section{Introduction}
-SIMBAD), NRAO, CXC, HEASARC, and ESA are the main
-organizations hosting raw and processed data from ground and space based
-telescopes.  These data archives also serve as the primary means for serving data
-to users when the data are taken in queue mode, i.e., when the data are taken
-while the observer is not on-site.
+Data sharing has taken on a variety of forms.  The most prominent are the
+major observatory archives: MAST, NOAO, ESO, ESA, IPAC, CDS, NRAO, CXC, HEASARC,
+and CADC are the main organizations hosting raw and processed data from
+ground and space based telescopes.  These data archives also serve as the
+primary means for serving data to users when the data are taken in queue
+mode, i.e., when the data are taken while the observer is not on-site.
 
 In addition to observatories and telescopes, individual surveys often share
 their full data sets.  In some cases, these data sets are shared via the
 observatory that acquired them, for example, the all-sky data acquired with
 Planck, WMAP, and COBE\@.  Other surveys, particularly ground-based surveys,
-serve their own data.  Examples include SDSS, 2MASS, UKIDSS, and likely many
-more in the near future.
+serve their own data.  Examples include SDSS, 2MASS, UKIDSS, VSA.
+
+%TODO: mention data collector sites, too, nasa\_exoplanet\_archive, simbad,
+%skyview, OAC, etc.
 
-Individual teams and small groups will often share their data via their own
-custom websites.  These services do not follow any particular standard and can
+Individual teams and small groups often share their data via their own
+custom websites, too.  These services do not follow any particular standard and can
 be widely varied in the type and amount of data shared.  Sometimes these data
 are shared via the archive systems (e.g., IRSA at IPAC hosts many individual
 survey data sets), while others use their own web hosting systems \citep[e.g.,
 MAGPIS;][]{Helfand2006}.
 
-Finally, there are other data types relevant to astronomy that are not served
-by the typical astronomical databases.  Examples include databases of molecular
-and atomic properties, such as those provided by
-Splatalogue\footnote{\url{http://www.splatalogue.net/}} and the NIST Atomic
-Spectra
-Database\footnote{\url{https://www.nist.gov/pml/atomic-spectra-database}}, or
-services that are computationally intensive or require constant updates, like
-Solar System ephemerides\footnote{e.g.,
-\url{https://ssd.jpl.nasa.gov/?ephemerides}}.  
-
-\astroquery arose from a desire to access these databases from the command line
+Finally, there are other data types relevant to astronomy that are not
+served by the typical astronomical databases.  Examples include databases of
+molecular and atomic properties, such as those provided by Splatalogue and
+the NIST Atomic Spectra Database, or services that are computationally
+intensive or require constant updates, like Solar System ephemerides
+provided by services like JPL HORIZONS, or the Minor Planet Center.
```

Comment:
Why did you remove the footnotes?

Reply:
ohh, sorry, it seems I never sent my comments. For this one I wrote: I don't think we need to do footnotes here, if yes, why there is no footnote for e.g. the mast archive or 2MASS, or any of the others above.